fn/operate/runner_pools.md

```diff
@@ -6,96 +6,183 @@ You can run a load-balanced setup for fn to route requests to a group of one or
 
 ## Starting the components (as regular processes)
 
+### Persisted storage
+
+For the particular example we'd use MySQL 5.7.22:
+
+```bash
+docker run --name fn-mysql \
+        -p 3306:3306 \
+        -e MYSQL_DATABASE=funcs \
+        -e MYSQL_ROOT_PASSWORD=root \
+        -d mysql:5.7.22
+```
+
+### DNS name requirement
+
+For the following deployment there's a need in the following DNS names:
+
+ - runner LB node DNS name: `lb.fn.local`
```

Comment:
it would be nice if this example 'just worked' by copying in the commands, if possible -- for users moving to prod deployments, this is easy enough to replace either way (ie if it was 127.0.0.1 or is lb.fn.local)

Reply:
Ok, I’m fine with it.